### ISSUE TITLE
Pipes connected directly to cooling tower don't work after a while

### ISSUE THREAD

**Issue by Memnoch72** [MaFi-Games/Captain-of-Industry-issues#427]
**Describe the issue**
I'm using 4 cooling towers to produce water from the output of a high-pressure turbine.  The ratio would be 24:24, and the output water is piped back into the system, so there is no backup of water.  The water is not flowing down the pipe.

If the line is run directly into the cooling towers, meaning that there isn't a 1 or more space gap between the pipe and the cooling tower, it will stop working after time.  It is one pipe connecting 4 high-pressure turbines that feed into the 4 cooling towers, again, a 1:1 ratio.  If the pipe is run one space out from the cooling tower, then a 1 (or more) length of pipe is connected to the tower, then there is no issue.  The steam will eventually back up and not continue on the pipe from the first cooling tower.

I think this may have happened in a previous game of mine on the output also, but I can't be 100% certain. 

**Expected behavior**
Steam should continue to run down the pipe

**To Reproduce**
Connect 4 cooling towers together with a pipe, without having any gaps between the pipe and the cooling tower

**Additional context**
v0.4.5b - most current version

**Attachments**
![This one works](https://user-images.githubusercontent.com/46352133/175747454-013e0643-4448-4e05-900d-6c80a0f5184f.jpg)
![This one doesn't work](https://user-images.githubusercontent.com/46352133/175747911-e6053928-1d3a-46c5-a651-2bfd179955ca.jpg)

**Comment by MrJoshua023** [comment]
Unique to 4 perhaps?  With 2 towers and thus only 1 connector, they work fine.
![image](https://user-images.githubusercontent.com/62954862/175772318-41435e0f-ec42-47b7-b20b-b6b4f984c745.png)

**Comment by dgzeij** [comment]
Check pipe balancer priority rules or try pause the smoke stack. The smoke stack is in use while the third cooling tower is not.

**Comment by MaFi-Games** [comment]
Please provide a save file

**Comment by MaFi-Games** [comment]
Does this still happen in version 0.4.6?

**Comment by JoeRoyle** [comment]
I would have to test it again, I fixed everything so I would have to put it
all back ;)

**Comment by MaFi-Games** [comment]
Closing as fixed. Feel free to file new ticket if you have new issue with this.

**Comment by Memnoch72** [comment]
I had the same type of issue with pipes running out of oil pumps, but I was panicked and fixed the issue before I ran out of diesel.  When the pipes were running in tandem, without a space out from the pump, the pipes eventually 'clogged' up and would not accept oil from two of the three pumps.  This only has happened when pipes were directly running in tandem from one pump to the other, exactly like my screenshot on the water outputs.  I do have a save game of before I fixed it thankfully, I will try to reproduce it later.
